### ISSUE TITLE
Abandoning in ranked before first blood doesn't cause mmr loss

### ISSUE THREAD

**Issue by Jason5613** [ValveSoftware/Dota2-Gameplay#16138]
### Description

People are abusing how abandon works in ranked. This incident mostly happens in high matchmaking rating. Griefers are taking advantage of the current state of the system as it forces other people who are in the lobby to abandon so that they may not lose MMR because of griefers. However, the griefers are not punished at all for their behavior as the game state is not counted so they are not able to be reported by other players for grieving and be punished by the system. This causes the griefers to keep continuing their behavior and ruining Dota 2 matchmaking without being punished. Please fix it so that abandoning before first blood still cause MMR loss.

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/165370959/dde1fefc-592c-4b1b-9cf1-95492835f8da)
https://stratz.com/players/173631721
Example of a griefer in SEA server. Running down mid for hundreds of games every time he does not get his role.

**Comment by commentator24** [comment]
im pretty sure the first one abandon (including their party members) get mmr loss but others not in that party do not get mmr change if its safe to leave.
unless they changed that? which i doubt

**Comment by Jason5613** [comment]
Yes they somehow changed it. The first one to abandon before first blood doesn't get mmr loss anymore.
